components/dropdown/README.md

```diff
@@ -1,7 +1,7 @@
 # Dropdowns
 A Dropdown is a button that opens a floating container to offer menu items or other content.
 
-<!-- docs: demo autoSize:false align:flex-start size:medium -->
+<!-- docs: demo align:flex-start autoSize:false size:medium -->
```

Comment:
I noticed this one isn't using `autoOpen` is that intended?

Reply:
Yup, this one has a more delayed opening defined in the demo script